docs-src/src/layouts/Navigation.js

```diff
@@ -1,9 +1,49 @@
-import React from 'react'
+import React, {useState} from 'react'
 import Menu from 'harmonium/lib/Menu'
 import Drawer from 'harmonium/lib/Drawer'
 import packageInfo from '../../../package.json'
+import Input from 'harmonium/lib/Input'
+import menuItems from '../pages/menuItems'
 
 export default function Navigation() {
+
+  //Use local state for controlled form
+  const [formValue, setFormValue] = useState('')
+
+  //Set local state to form value
+  const handleFormChange = (e) => {
+    setFormValue(e.target.value)
+  }
+
+  //Check if item is including in search form
+  function itemCheck(item){
+    return item.label.toLowerCase().includes(formValue.toLowerCase());
+  }
+
+  //Display filtered menu items 
+  function displayMenu(menu){
+    return menu.map( (cat, i) => {
+      if (cat.contents.filter( (item) => itemCheck(item)).length >= 1)
+      return ( 
+        //Category name
+        <Menu vertical key={i}>
+          <Menu.Item text>
+            <h5>{cat.category}</h5>
+          </Menu.Item>
+          {cat.contents.map( (item, j) => {
+          if (itemCheck(item)){
+            return (
+            //Link to component
+            <Menu.Item key={j}>
```

Comment:
Its not ideal to use indexes of the iteration as keys because it messes with react's method of reconciliation. Probably better to use `item.label`